mail/local-mail/src/main/kotlin/com/ritense/mail/service/LocalMailDispatcher.kt

```diff
@@ -0,0 +1,20 @@
+package com.ritense.mail.service
+
+import com.ritense.mail.MailDispatcher
+import com.ritense.valtimo.contract.mail.model.MailMessageStatus
+import com.ritense.valtimo.contract.mail.model.RawMailMessage
+import com.ritense.valtimo.contract.mail.model.TemplatedMailMessage
+
+class LocalMailDispatcher : MailDispatcher {
+    override fun send(rawMailMessage: RawMailMessage): MutableList<MailMessageStatus> {
```

Comment:
Misschien neit iets voor deze PR, maar waarom is dit een MutableList? Het gaat toch om het resultaat van een mail component? Wat wil je daar dan aan kunnen wijzigen?